courtroom.cpp

```diff
@@ -980,15 +980,36 @@ void Courtroom::handle_chatmessage_3()
   QString f_emote = m_chatmessage[EMOTE];
 
   ui_vp_showname_image->show();
+
+  // Check for any of 9 possible ways that showname images are
   QVector<QString> exts = {".png", ".jpg", ".bmp"};
+  // 3 places (in order)
+  // 1. Variant folder in character folder in theme folder
+  // 2. Character folder in folder folder
```

Comment:
Folder folder folder folder folder folder.

Reply:
Do you think example directories would be better as comment?

Reply:
Well, from what I can gather, the actual paths being looked up here are:

1. `base/themes/<current_theme>/variants/<current_variant>/characters/<current_char>/showname.<ext>`
2. `base/themes/<current_theme>/characters/<current_char>/showname.<ext>`
3. `base/characters/<current_char>/showname.<ext>`

So the comments should better indicate these. Example:

```cpp
// 1. Character folder in variant folder in theme folder
// 2. Character folder in theme folder
// 3. Character folder
```

Reply:
Oops, leftover from old PR, will fix